db/migrate/20210208145457_create_comments.rb

```diff
@@ -0,0 +1,12 @@
+class CreateComments < ActiveRecord::Migration[6.1]
+  def change
+    create_table :comments do |t|
+      t.string :commenter
```

Comment:
[required] `t.string :commenter` this should be a `user` references, as you have another migration for this user references, so this field sounds of no valuable meaning. I would be better to remove it